src/ol/source/cartodb.js

```diff
@@ -0,0 +1,133 @@
+goog.provide('ol.source.CartoDB');
+
+goog.require('goog.asserts');
+goog.require('ol.source.State')
```

Comment:
Looks like a missing semicolon.  (We could make the linter warn here so we're not left with [confusing build errors](https://travis-ci.org/openlayers/ol3/builds/119613653).)